lib/src/base-styles/_nx-typography.scss

```diff
@@ -98,3 +98,10 @@
   line-height: 18px;
   margin-bottom: 0;
 }
+
+button {
```

Comment:
Global bare-element selectors are forbidden to allow compatibility with existing projects

Reply:
> Global bare-element selectors are forbidden to allow compatibility with existing projects

https://github.com/sonatype/sonatype-react-shared-components/pull/59/commits/67a7617c53ee049ea6c5cd4d1c407c69e4170039